inc/breadcrumbs.php

```diff
@@ -1032,7 +1032,13 @@ protected function add_post_terms( $post_id, $taxonomy ) {
 		if ( $terms && ! is_wp_error( $terms ) ) {
 
 			// Sort the terms by ID and get the first category.
-			usort( $terms, '_usort_terms_by_ID' );
+			if ( function_exists( 'wp_list_sort' ) ) {
+				$terms = wp_list_sort( $terms, 'term_id' );
+			}
+			else {
```

Comment:
According to the [WP coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style) please push the `else` to upper line so its formatted like `} else {`